src/types/profile-derived.js

```diff
@@ -65,13 +65,14 @@ export type CallNodeInfo = {
  */
 export type CallNodePath = IndexIntoFuncTable[];
 
-export type TracingMarker = {
+export type TracingMarker = {|
   start: Milliseconds,
   dur: Milliseconds,
   name: string,
   title: string | null,
   data: MarkerPayload,
-};
+  incomplete?: boolean,
```

Comment:
I like this, much better than my previous hacky solution of having a negative duration.
But it's used only for markers with no start, do you think we could use for markers with no end as well?

Reply:
Yes, that worked fine :-)